### PR TITLE
Update scenario1

### PR DESCRIPTION
**Pull request by cafriiseoul**
i updated scenario1 file

### CODE REVIEW

File: scenario1 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
 It is test file for scenario 1.
-
+cafriiseoul added this line.
```
Comment (cafrii) [inline scenario1]: suggest using quote around cafriiseoul.

File: scenario1 [hunk 2]
```diff
@@ -1,5 +1,5 @@
 
 It is test file for scenario 1.
```
Comment (cafrii) [inline scenario1:2]: LGTM

**Review by cafrii — DISMISSED** [review]
i added two comments. plz check them.

File: scenario1 (hunk 2 above)
Comment (cafriiseoul) [reply scenario1:2]: thnx

File: scenario1 (hunk 1 above)
Comment (cafriiseoul) [reply scenario1]: ok. try to edit by yourself.

Comment (cafrii) [reply scenario1]: i did it.

**Review by cafrii — COMMENTED** [review]
LGTM

**Review by cafrii — APPROVED** [review]
i approve

**Comment by cafrii** [comment]
merge with rebase

**Pull request merged**